classes/module/Module.php

```diff
@@ -758,7 +758,11 @@ public function enable($force_all = false)
             }
         }
 
-        if ($this->getOverrides() != null) {
+        if (count($items) == Shop::getTotalShops()) {
+            $force_all = true;
```

Comment:
@LittleBigDev you are right but your sentence is not

> As enabling/disabling a module only affect its behavior, and has nothing to do with overrides, there is no solution to your current issue.

 because enabling/disabling a module will install/uninstall its overrides [https://github.com/PrestaShop/PrestaShop/blob/develop/classes/module/Module.php#L867](url)
This is not correct because it will cause the malfunction of the module on the other shop (on which the module is installed).

I think that lines 761-770 and 866-868 should be totally removed.